_health-care/_js/components/personal-information/Gender.jsx

```diff
@@ -1,45 +1,42 @@
 import React from 'react';
 import _ from 'lodash';
 
+/**
+ * Select component for getting a person's gender.
+ *
+ * No validation is provided since base UI elements provide all
+ * common options as provided in reference PDF/JS form.
+ */
 class Gender extends React.Component {
   constructor() {
     super();
-    this.state = { hasError: false };
     this.handleChange = this.handleChange.bind(this);
   }
 
   componentWillMount() {
-    this.id = _.uniqueId();
+    this.selectId = _.uniqueId('gender-select-');
   }
 
+  // handleChange(domEvent) {
   handleChange() {
-    const gender = this.refs.gender.value;
-
-    if (this.validate(gender)) {
-      this.setState({ hasError: false });
-    } else {
-      this.setState({ hasError: true });
-    }
-
-    this.props.onUserInput(gender);
+    this.props.onUserInput(this.refs.gender.value);
   }
 
   render() {
-    const errorClass = this.state.hasError ? 'usa-input-error' : '';
     return (
       <div>
-        <div className={`usa-input-grid usa-input-grid-large ${errorClass}`}>
-          <label htmlFor={`${this.id}-gender`}>
+        <div>
+          <label htmlFor={this.selectId}>
             Gender
           </label>
           <select
-              id={`${this.id}-gender`}
+              id={this.selectId}
+              onChange={this.handleChange}
               ref="gender"
-              value={this.props.gender}
-              onChange={this.handleChange}>
-            <option value="0"></option>
-            <option value="M">Male</option>
+              value={this.props.optionState}>
```

Comment:
What is `optionState`? I don't see it being passed as a prop on `Gender` in `NameAndGeneralInfoSection.jsx`.